website/docs/requests.md

```diff
@@ -7,85 +7,104 @@ A request is counted every time your application downloads a configuration file
 CDN. The frequency of these downloads is totally under your control. Between downloads you can access your feature
 flags as many times as you like, it still counts as one.
 
-Keep track of the number of requests your apps are making on the <a href="https://app.configcat.com/product/statistics" target="_blank">ConfigCat Dashboard.</a> 
+Keep track of the number of requests your apps are making on the <a href="https://app.configcat.com/product/statistics" target="_blank">statistics page.</a> 
 
 ![ConfigCat Statistics](/assets/stats.png)
 
-Check out our <a href="https://configcat.com/architecture" target="_blank">architectural overview</a>.
+Here is an <a href="https://configcat.com/architecture" target="_blank">overview of the ConfigCat architecture</a>.
 
-## Current request/month limits
-| Plan          | requests/month | requests/second |
-| ------------- | -------------: | --------------: |
-| **Free**      |      5,000,000 |              ~2 |
-| **Pro**       |     25,000,000 |             ~10 |
-| **Unlimited** |    250,000,000 |            ~100 |
-| **Dedicated** |      unlimited |      contact us |
+# Current request number limits
+## Shared infrastructure
+The following plans run on shared infrastructure. So all customers use the same API nodes and Config Delivery Network (CDN).
+
+| Plan           | total req/month | avg. req/sec | max. spike req/s |
+| -------------- | --------------: | -----------: | ---------------: |
+| **Free**       |       5,000,000 |           ~2 |               ~4 |
+| **Pro**        |      25,000,000 |          ~10 |              ~20 |
+| **Smart**      |     250,000,000 |         ~100 |             ~200 |
+| **Enterprise** |   1,000,000,000 |         ~400 |             ~800 |
 
 :::info
 If you hit this limit, we will keep your application up and running. However, you can expect us contacting you on how we can meet your needs.
 :::
 
+## Dedicated infrastructure
+The following plans include dedicated API and CDN nodes.
+### Hosted
+Runs on dedicated servers provided by ConfigCat.
+The basic package includes:
+- 1x API node
+- 6x CDN nodes (3 pairs, 1 pair in every location)
```

Comment:
az 1 pair in every location  részt annyira nem értem. megmagyarázod pls?

Reply:
removed ;)